### PR TITLE
LVRUBYM-312: Html sceleton in index.html

### PR DESCRIPTION
**Pull request by Mykhailo123**
## JIRA

* [Main JIRA ticket](http://ssu-jira.softserveinc.com/browse/LVRUBYM-311)
* [JIRA subtask](http://ssu-jira.softserveinc.com/browse/LVRUBYM-312)

## Code reviewers

- [x] @vikabelous
- [x] @HelenMaherovska
- [x] @JulkaIII

### Second Level Review

- [ ] @4asof4ik

## Summary of issue

Base template. Load angular material

## Summary of change

- add components navbar, header

## Testing approach

Test with JSCS

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: app/js/app.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
 var angular = require('angular'),
+    generalModule = require('./general/general.module.js'),
     mainModule = require('./main/main.module.js'),
     vendorModule = require('./requirements.js');
 
 angular
   .module('school_area', [
+         'ngMaterial',
```
Comment (vikabelous) [inline app/js/app.js]: Add `ngMaterial` to the `vendor` module

File: app/js/main/components/main.template.html [hunk 2]
```diff
@@ -1,4 +1,94 @@
...
+      <ul>
+  <li class="journal">
+    <a ui-sref="main">
+      Journal
+    </a>
+  </li>
+  <li>
+    <ul>
+      <li ui-sref-active="main" class="active">
+        <a ui-sref="main">
```
Comment (4asof4ik) [inline app/js/main/components/main.template.html]: ui-sref="main"  what is that ?

File: app/js/main/main.module.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 var mainComponent = require('./components/main.component.js'),
     mainRoute = require('./main.route.js');
 require('../../css/style.scss');
+require('angular-material/angular-material.css');
```
Comment (4asof4ik) [inline app/js/main/main.module.js]: Please add angular material as I say before

Comment (Mykhailo123) [inline app/js/main/main.module.js]: It doesn't support styles without this line.

File: app/js/main/components/main.component.js [hunk 4]
```diff
@@ -1,10 +1,39 @@
...
-    controller: MainController
+    controller: [
+        '$scope',
+        MainController
+    ],
+    transclude: true
   });
 
-function MainController() {
+function MainController($scope) {
```
Comment (Vovk-Nazar) [inline app/js/main/components/main.component.js]: please use $scope or this

File: app/js/main/components/main.component.js [hunk 5]
```diff
@@ -1,10 +1,39 @@
...
   var ctrl = this;
+  var date = $scope;
+  var month = $scope;
+  var day = $scope;
+
+  date.selectedDate = new Date();
```
Comment (Vovk-Nazar) [inline app/js/main/components/main.component.js]: for date use momentjs

File: app/js/main/components/main.component.js [hunk 6]
```diff
@@ -1,10 +1,39 @@
...
+  month.monthNames = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'June',
+  'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
+  day.dayNames = ['Mon','Tue','Wed','Thu','Fri','Sat','Sun'];
+
+  date.nextDate = () => {
+    date.selectedDate.setDate(date.selectedDate.getDate() + 1);
```
Comment (Vovk-Nazar) [inline app/js/main/components/main.component.js]: why you use ES6

File: app/js/main/main.module.js [hunk 7]
```diff
@@ -1,9 +1,11 @@
 var mainComponent = require('./components/main.component.js'),
-    mainRoute = require('./main.route.js');
+    mainRoute = require('./main.route.js'),
+    presenceModule = require('./../presence/presence.module.js');
```
Comment (Vovk-Nazar) [inline app/js/main/main.module.js]: http://stackoverflow.com/questions/694102/declaring-multiple-variables-in-javascript

File: app/js/main/components/main.component.js [hunk 8]
```diff
@@ -1,10 +1,39 @@
 module.exports = angular
-  .module('school_area.main.component', [])
+  .module('schoolArea.main.component', [])
   .component('mainComponent', {
     templateUrl: './app/js/main/components/main.template.html',
-    controller: MainController
+    controller: [
+        '$scope',
```
Comment (andriyarterchuk) [inline app/js/main/components/main.component.js]: we decided not to use a $scope, isn't it?

File: app/js/main/main.route.js [hunk 9]
```diff
@@ -1,10 +1,10 @@
 module.exports = angular
-  .module('school_area.main.route', [])
+  .module('schoolArea.main.route', [])
```
Comment (Vovk-Nazar) [inline app/js/main/main.route.js]: example module naming you can find in skype

File: app/js/presence/presence.module.js [hunk 10]
```diff
@@ -0,0 +1,7 @@
+var presenceComponent = require('./components/presence.component.js'),
+    presenceRoute = require('./presence.route.js');
+module.exports = angular
```
Comment (Vovk-Nazar) [inline app/js/presence/presence.module.js]: add empty line

File: app/js/presence/presence.route.js [hunk 11]
```diff
@@ -0,0 +1,9 @@
+module.exports = angular
+  .module('schoolArea.presence.route', [])
+  .config(function($stateProvider, $urlRouterProvider) {
```
Comment (Vovk-Nazar) [inline app/js/presence/presence.route.js]: $urlRouterProvider where you use this

File: app/js/presence/presence.route.js [hunk 12]
```diff
@@ -0,0 +1,9 @@
...
+    $stateProvider
+    .state('main.presence', {
```
Comment (Vovk-Nazar) [inline app/js/presence/presence.route.js]: add tab

File: app/js/main/components/main.component.js (hunk 4 above)
Comment (Mykhailo123) [inline app/js/main/components/main.component.js]: Ok

File: app/js/main/components/main.component.js (hunk 5 above)
Comment (Mykhailo123) [inline app/js/main/components/main.component.js]: Ok

File: app/js/main/components/main.component.js (hunk 6 above)
Comment (Mykhailo123) [inline app/js/main/components/main.component.js]: Changed

**Pull request merged**